migrations/2_deploy_ingress_contract.js

```diff
@@ -1,23 +1,35 @@
-var Ingress = artifacts.require("./Ingress.sol");
+const Web3Utils = require("web3-utils");
+
+const Rules = artifacts.require("./Rules.sol");
+const Ingress = artifacts.require("./Ingress.sol");
+
+// "rules" as bytes32
+const rulesContractName = Web3Utils.utf8ToHex("rules");
 
 /* The address of the ingress contract if pre deployed */
 var ingressAddress = process.env.INGRESS_CONTRACT_ADDRESS;
 
-module.exports = (deployer, network) => {
+module.exports = async(deployer, network) => {
     if (! ingressAddress) {
         // Only deploy if we haven't been provided a predeployed address
         deployer.deploy(Ingress).then(() => {
```

Comment:
I think the error you're getting out of CI might be due to this still being a promise style call when the function is marked async. The test is racing ahead before the Ingress has finished deploying